src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,42 @@
+package racingcar.view;
+
+import java.util.Scanner;
+import racingcar.domain.dto.ParticipantsRequestDto;
+import racingcar.domain.dto.TurnRequestDto;
+
+public class InputView {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static final String NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+    private static final String TURN_MESSAGE = "시도할 횟수는 몇 회인가요?";
+    private static final String DELIMITER = ",";
+    private static final String ERROR_LOG = "빈 공백은 들어올 수 없습니다.";
```

Comment:
`클래스 변수`와 상수의 순서는 (위) 상수 -> 클래스 변수 -> 인스턴스 변수 (아래) 순서대로 존재해야 해요! 😃 
또한 이 변수들끼리는 한 줄 띄어서 가독성을 높여주면 좋을 것 같아요. 🤗 

```suggestion
    private static final String NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
    private static final String TURN_MESSAGE = "시도할 횟수는 몇 회인가요?";
    private static final String DELIMITER = ",";
    private static final String ERROR_LOG = "빈 공백은 들어올 수 없습니다.";
    
    private static Scanner scanner = new Scanner(System.in);
```